src/plugins/data_view_field_editor/public/components/field_editor/field_editor.tsx

```diff
@@ -189,6 +152,89 @@ const FieldEditorComponent = ({ field, onChange, onFormModifiedChange }: Props)
 
   const isValueVisible = get(formData, '__meta__.isValueVisible');
 
+  const lastPreview$ = useMemo(() => {
+    const replaySubj = new BehaviorSubject<FieldPreview[]>([]);
+    fieldPreview$.subscribe(replaySubj);
```

Comment:
Could we make `fieldPreview$` a BehaviourSubject in the first place so that we don't need to sync its state to other places and instead use it when we need it with `getValue()`

Also note: if we call `subscribe()` somewhere, most likely we need to unsubscribe explicitly, otherwise, there is a memory leak.